packages/clarity-tutorials/src/clients/rocketTutorial/rocketMarket.ts

```diff
@@ -7,13 +7,13 @@ export class RocketMarketClient extends Client {
   async balanceOf(owner: string): Promise<number> {
     const query = this.createQuery({ method: { name: "balance-of", args: [`'${owner}`] } });
     const res = await this.submitQuery(query);
-    return parseInt(res.data.result);
+    return parseInt(res.result!);
   }
 
-  async ownerOf(tokenId: number): Promise<number> {
+  async ownerOf(tokenId: number): Promise<string> {
     const query = this.createQuery({ method: { name: "owner-of", args: [`${tokenId}`] } });
     const res = await this.submitQuery(query);
-    return res.data.result.replace(/'/g, "");
+    return res.result!.replace(/'/g, "");
```

Comment:
Should we safely unwrap our results instead?

Reply:
Got all instances of these force casts fixed. Getting generic type interface working for loosely typed result-like objects took longer than expected. All good now.